rs/nns/governance/tests/governance.rs

```diff
@@ -11399,7 +11399,7 @@ lazy_static! {
         let mut result = vec![
             sns_swap_pb::CfParticipant {
                 controller: Some(principal(1)),
-                hotkey_principal: principal(1).to_string(),
+                hotkey_principal: String::new(),
```

Comment:
I'm thinking how to best help the clients whom we potentially break this way. 

What if we set this non-optional string to something like 

```
Field `hotkey_principal` is obsolete as a misnomer, as it used to hold the *controller* principal ID of the (Neuron's Fund-participating) NNS neuron, and not NNS neuron hotkeys. Please use field `controller` instead for the NNS neuron controller.
```

Reply:
This is only used in tests, so I think we won't break anyone with the line you commented on. However, I will do this in swap